guide/13-managing-arcgis-applications/hub-premium-guide.ipynb

```diff
@@ -0,0 +1,471 @@
+{
```

Comment:
<p>ArcGIS Hub Premium enables organizations, like local non-profits, governments, or schools, to share a project or update with their community by starting an initiative or creating a site and organizing content and teams to share web maps, apps, datasets, and more with the public. Hub Premium also supports creating and sharing events related to your initiative using hub's built-in events management system. Click <a href="https://www.esri.com/en-us/arcgis/products/arcgis-hub/resources" target="_blank">here</a> to learn more about ArcGIS Hub. </p><p>In this notebook, we will explore the Hub Premium capabilities of <strong>Initiatives</strong> and <strong>Events</strong>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>